skimage/measure/tests/test_marching_cubes.py

```diff
@@ -4,6 +4,7 @@
                              mesh_surface_area, correct_mesh_orientation)
 from skimage._shared import testing
 from skimage._shared.testing import assert_array_equal
+from skimage._shared._warnings import expected_warnings
```

Comment:
Could you revert this commit, please? Otherwise, good-to-go.

Reply:
done!

Reply:
Sorry for the delay. Done!